ontologies/vocabs.ttl

```diff
@@ -1188,8 +1175,10 @@ Tag for the healthcare industry sector"""^^xsd:string ;
 
 gist:_IndustrySector_hospitality-Leisure
 	a gist:IndustrySector ;
-	rdfs:label "Hospitality Leisure"^^xsd:string ;
-	skos:altLabel "Hospitality-Leisure"^^xsd:string ;
+	skos:altLabel
+		"Hospitality Leisure"^^xsd:string ,
+		"Hospitality-Leisure"^^xsd:string
```

Comment:
Is this helpful?
Also, should there not be a stixTerm? I guess that is a different git issue?